aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+- Rename the downloaded file to `my_app.aiconfig.json`.
+- Replace `my_app.aiconfig.json` in the space repo with the downloaded one.
+
+Anyone who visits your space will see the state represented by `my_app.aiconfig.json`.
+
+## Examples
+
+- [Music Playground](https://huggingface.co/spaces/lastmileai/music_generator). Make your demo more captivating with different modalities.
+- [AI Story Generator](https://huggingface.co/spaces/lastmileai/ai_story_generator). Easy to chain models for a workflow like story building.
+- [Stable Diffusion XL Playground](https://huggingface.co/spaces/lastmileai/sdxl_playground). Create interactive prompt templates for your model.
+
+## Features
+
+### Gradio Workbook Structure
+
+Gradio Workbooks are inspired by Jupyter notebooks. Each workbook is made up of cells. The cell is associated with a model and running the cell executes your prompt on that model.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/85c9953f-0cf6-4d0c-84bf-5fffa4e5bcf5" width="800"/>
+  </picture>
+</div>
+
+### Parameters
+
+Parameters are variables that you can define and pass into any cell within your workbook. You can set both **global parameters** and **local parameters**. For example, you can define parameters such as language or product in a cell like this with the handlebars syntax as shown below: `"Write a poem in {{language}} about a salesman selling {{product}}."`
+
+#### Global Parameters
+
+You can set global parameters for your entire workbook. Click on **Global Parameters {}** on the top of your workbook and set your global parameters that can be used in any cell of the workbook.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/631f33ac-cf66-4e40-a7aa-c28d37b2ffad" width="800"/>
+  </picture>
+</div>
+
+#### Local Parameters
+
+You can set local parameters that only apply to the cell you are in. Click on the cell sidebar and **Local Parameters**. Note: Local parameters will override the global parameters if they have the same name.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/0935ac7c-5e68-42f1-befe-e279b027cac8" width="800"/>
+  </picture>
+</div>
+
+### Chaining
+
+You can reference the output from one cell in another cell. This allows you to create complex, interconnected workflows within your workbook.
+
+The summarization prompt below uses the output of the story_gen prompt using handlebars syntax
+(e.g `{{story_gen.output}}`)
+
+![chaining](https://github.com/lastmile-ai/aiconfig/assets/81494782/be65e264-3734-422e-9967-35a44d77cbe7)
+
+### Download
+
+Click **Download** to download the `aiconfig` file that captures the current state of your space in a JSON file. Example Space and corresponding `aiconfig` file:
```

Comment:
"Click **Download** to download the `aiconfig` file that captures the current state of your space"